ansible/roles/machine/provision_ipaserver/tasks/main.yml

```diff
@@ -5,6 +5,105 @@
     dest: /usr/local/bin/ipa-run-webui-tests
     mode: 0755
 
+- block:
+    - name: set PKI related variables
+      set_fact:
```

Comment:
`set_fact` defines global variables which should be avoided unless absolutely necessary. 
Since the following variables are used only inside this block please consider using `vars` section instead

Reply:
@wladich Thank you for your feedback, change is implemented.